ChangeLog

```diff
@@ -3,6 +3,19 @@ Pylint's ChangeLog
 ------------------
 
 
+What's New in Pylint 2.15.0?
+============================
+Release date: TBA
+
+..
+  Put new features here and also in 'doc/whatsnew/2.15.rst'
+
+
+* Emit ``modified-iterating-list`` and analogous messages for dicts and sets when using the ``del`` keyword.
```

Comment:
I'd be fine with putting this in `2.14`. Doesn't seem like likely to regress. Knock on wood...

Reply:
I guess I'd prefer not to add things to something we pre-released. And it's not ready anyway, since there's a failing test to look into.